.gitignore

```diff
@@ -93,13 +93,6 @@ _build
 /manual/src/cmds/warnings-help.etex
 /manual/src/warnings-help.etex
 
-/api_docgen/man
-/api_docgen/html
-/api_docgen/build
-/api_docgen/latex
-/api_docgen/texi
-/api_docgen/Compiler_libs.mld
-
```

Comment:
Why are these moving to a separate `.gitignore`? For better or worse, the policy is a top-level one.